Fenced_Frames_Ads_Reporting.md

```diff
@@ -159,8 +166,27 @@ Currently, the only `eventType` that `setReportEventDataForAutomaticBeacons` all
 
 If invoked multiple times, the latest invocation before the top-level navigation would be the one that’s honored.
 
-`eventData` can be empty, in which case the automatic beacon will still be sent but without an event data body in the HTTP request.
+`eventData` is optional, and can be empty. If `eventData` is not specified, or is empty, the automatic beacon will still be sent but without an event data body in the HTTP request.
 
 If `setReportEventDataForAutomaticBeacons` is not invoked, the browser will not send an automatic beacon because the `destination` is unknown.
 
+# Support for Ad Components
+## Goal
+When a rendered ad is composed of [multiple pieces](https://github.com/WICG/turtledove/blob/main/FLEDGE.md#34-ads-composed-of-multiple-pieces), it is useful to detect user clicks that happened on ad components.
+
+## Design
+### Event Type and Reporting Destination
+For fenced frames rendering the ad components under the top-level ad fenced frame, the `reserved.top_navigation` event type and corresponding reporting destination registered for the top-level fenced frame are reused when beacons are sent from the ad component fenced frames.
```

Comment:
What does "reused" mean in this case?  Does it mean that the beacons are sent to these destinations when the events happen in the component frames?  Does this mean they're duplicated between the top and component frames?